checkout-extension/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,35 @@
+# Learn more about configuring your checkout UI extension:
+# https://shopify.dev/api/checkout-extensions/checkout/configuration
+
+name = "{{ name }}"
```

Comment:
one more request, please move this field under [[extensions]]
We no longer support top-level names